pkg/controller/installation/installation_controller_test.go

```diff
@@ -206,7 +206,7 @@ func TestCreateInstallationCR_installationTypeDefault(t *testing.T) {
 	mockClient := fake.NewFakeClientWithScheme(buildScheme())
 	ctx := context.TODO()
 
-	installationType := "managed"
+	installationType := "managed-api"
```

Comment:
Should this stay on managed too then?

Reply:
I think it's ok for the tests to default to managed-api, after 2.7.1 we should not be doing many releases for RHMI.

Reply:
Hmm, though actually, do the tests pass like this? I feel like a bunch will fail due to missing namespaces, etc.

Reply:
@philbrookes They've been split based on type and further refined here:
https://issues.redhat.com/browse/MGDAPI-46

Reply:
We have the pipeline in place so we will monitor the tests daily. There was just one failure last night.

I just think it is a little strange for the tests defaulting to 'managed-api' whereas the operator itself defaulting to 'managed-api'. But I am ok with that.